basic/README.md

```diff
@@ -18,3 +18,251 @@ function.
     open http://localhost:3000/
 
 Runs the app from http://localhost:3000
+
+For more information see [Use Mapbox GL JS in a React app](https://docs.mapbox.com/help/tutorials/use-mapbox-gl-js-with-react/) 
+
+
+Documentation
```

Comment:
Can this documentation actually be moved into a ticket? I can followup with the documentation team at Mapbox to make these changes!

Reply:
What do you mean with "Can this documentation actually be moved into a ticket?". I think this page have to be updated if we change the examples: https://docs.mapbox.com/help/tutorials/use-mapbox-gl-js-with-react/

Reply:
@astridx I agree! I just think the text for this should not be written in `basic/README.md`. I'm suggesting we pull this out into a [new ticket](https://github.com/mapbox/mapbox-react-examples/issues/new) with a title that says "Update the documentation with this text to account for new changes"?